cypress/integration/api/book.test.ts

```diff
@@ -0,0 +1,129 @@
+import { HttpMethod } from "@lib/http-method";
+import { StatusCodes } from "http-status-codes";
+import {
+  TEST_BOOK_UUID,
+  TEST_BOOK_1_ISBN,
+  TEST_BOOK_2_ISBN,
+  TEST_COURSE_UUID,
+} from "../../support/constants";
+
```

Comment:
One other test that might be worthwhile is testing that if you pass in a string that isn't an integer or is negative to `length` or `page` that it gives a "bad request" response.

Reply:
Great idea! But, currently, we just return default values of length = 20 and page = 0 if an invalid string is added (should be extended to negative integers too).

I think it makes more sense to return default values rather than giving an error for both cases?

Reply:
Okay, yes the current behaviour makes sense to me. I think it might still be worthwhile to test passing incorrect types to the API

Reply:
Sounds good. Will do!